README.md

```diff
@@ -37,26 +46,7 @@ Operators can behave like managed service providers. Their user interface on the
 
 ## Getting Started 
 
-### Installation
-
-Install OLM on a Kubernetes or OpenShift cluster by following the [installation guide].
-
-For a complete end-to-end example of how OLM fits into the Operator Framework, see the [Operator Framework Getting Started Guide](https://github.com/operator-framework/getting-started). Also, see [Getting Started on OperatorHub.io](https://operatorhub.io/getting-started).
-
-**NOTE:** OLM is installed by default in OpenShift 4.0 and above.
-
-## User Interface (Running the console Locally)
```

Comment:
This section is missing from the Getting Started doc

Reply:
It will be in https://github.com/operator-framework/olm-docs/pull/122